src/Dependencies/Roslyn/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09"
+    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09",
+    "System.ValueTuple": "4.0.1-beta-24425-02"
```

Comment:
Note that only uses it in Interactive Host. We do not have it in DevEnv process. We should probably add it since we have other features that need it.

However, note that adding a new binary might trigger RPS perf gates.